src/core/session.cpp

```diff
@@ -207,6 +217,33 @@ void session::reset_after_move()
     pool_ = nullptr;
 }
 
+schema_table_name* session::alloc_schema_table_name(std::string & tableName)
+{
+    std::size_t dot_pos = tableName.find( '.' );
```

Comment:
Out of curiosity, what happens if there is more than one period in the string?

Also, what if it's in the first or last position?

Should we validate it here somehow, maybe?

Reply:
The table name will be treated as if it contains a ".". I was thinking about it, and came to the conclusion that - well - if you try to look up a table name with a dot - which typically isn't allowed in SQL table names. (I don't know of anything allowing it) - It simply won't find anything... So, garbage in, garbage out. It surely shouldn't break anything.